src/App.js

```diff
@@ -1,39 +1,64 @@
 import React, { Component } from 'react'
+import axios from 'axios'
 import Movie from './components/Movie'
+import Header from './components/Header'
 import MovieList from './components/MovieList'
 import SearchBar from './components/SearchBar'
-import movies from './movies.js'
+
+const DEFAULT_STATUS = 'Search a massive database of movies and TV shows!'
+const SearchStatus = ({text}) => (<h1>{text}</h1>)
 
 class App extends Component {
   constructor() {
     super()
     this.state = {
-      movies,
-      searchField: ''
+      movies: [],
+      search: '',
+      status: DEFAULT_STATUS
     }
-    this.onSearchChange = this.onSearchChange.bind(this)
+    this.handleSearchChange = this.handleSearchChange.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
   }
 
-  onSearchChange(e){
-    this.setState({ searchField: e.target.value })
+  fetchMovies(search){
+    const { movies, status } = this.state
+    let typedSearch = `http://www.omdbapi.com/?apikey=df1379a2&s=${search}`
+    axios.get(typedSearch)
+    .then(result=> {
+      return result.data.Error ? this.setState({status: result.data.Error})
+      : this.setState({movies: result.data.Search})
+    })
   }
 
-  filterMoviesBy({ movies, searchField }){
-    return movies.filter(movie =>
-       movie.title.toLowerCase().includes(searchField.toLowerCase())
-    )
+  handleSearchChange(e){
+    this.setState({ search: e.target.value, status: DEFAULT_STATUS, movies: [] })
   }
 
+  handleSubmit(e){
+    e.preventDefault()
+    const { search, status } = this.state
+    this.setState({ status: '' })
```

Comment:
Why clear the search query on submit? Search bars usually leave it until you clear it to search again.

Reply:
This doesn't clear the search query, just sets the `status` to an empty string so that the message inviting the user to search goes away on submit